src/store/metadata/stub/content/subtopics.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
we shouldn't need this file, as a sub topic is also a thing from the topic level (and because this is our source of truth for seed RDF) we'll end up loading the same thing into oxigraph twice, will get very confusing if the definitions diverge.